js/menu.js

```diff
@@ -840,6 +844,29 @@ let referencePane = {
     this.$tableBody = newBody;
     this.$table.appendChild(this.$tableBody);
   },
+
+  dragStart(pointerDownEvent) {
+    let startingMousePos = pointerDownEvent.clientY;
+    let startingHeight = parseInt(getComputedStyle(this.$container).height);
+    let moveListener = pointerMoveEvent => {
+      if (pointerMoveEvent.buttons === 0) {
+        removeListeners();
+        return;
+      }
+      let desiredHeight = startingHeight - (pointerMoveEvent.clientY - startingMousePos);
+      this.$container.style.height =
+        Math.max(50, Math.min(visualViewport.height - 150, desiredHeight)) + 'px';
```

Comment:
This clamping should probably just be handled by CSS `min-height` and `max-height`.

Reply:
Done.